### ISSUE TITLE
JAWS reportedly fail to handle web-component based select component

### ISSUE THREAD

**Issue by christophe-g** [FreedomScientific/standards-support#776]
### Summary

We have received reports that Material Design webcomponent  (that we use for [accessiblesurveys.com](https://accessiblesurveys.com/)) is not behaving as expected when used in conjunction with JAWS screen reader. 

Test run with Orca screen reader on Linux pass.

Steps to reproduce: 
https://lit.dev/playground/#gist=8266d1cf3ac2739e43a64cfa77bc02bc

### Expected result

- Options are announced when they receive focus (after the listbox menu is opened). JAWS should announce: "blank", "apple", "orange", "banana" as for the native select component.
- The selection menu closes when a user select a value. (enter or space key)

### Actual result

The reported behavior is that: 

- Options are not announced when they receive focus
- The selection menu stays open once a user has selected a value.
- The selection menu needs to be closed manually (by pressing the escape key for instance)

### Example

https://lit.dev/playground/#gist=8266d1cf3ac2739e43a64cfa77bc02bc

### Additional Information

I cannot confirm the behavior as I do not have access to JAWS nor windows. 

The accessibility tree for the component is totally legit.

#### JAWS version and build number

JAWS 2023 (build number unknown)
 
#### Operating System and version

Windows (version unknown)

#### Browser and version:

Chrome latest

**Comment by JAWS-test** [comment]
The problem does not occur with Chrome 118.0.5993.118 and JAWS 2023.2307.37

**Comment by christophe-g** [comment]
Ok - thanks a lot @JAWS-test  for this rapid feedback - truly appreciated.

**Comment by stevefaulkner** [comment]
Thanks @JAWS-test, closing
